doc/user/olm-catalog/operator-bundles.md

```diff
@@ -0,0 +1,23 @@
+# Operator Bundles
+
+This document gives an overview of using `operator-sdk` to work with Operator [bundles][registry-bundle], namely on-disk bundle directories and creating bundle [images/metadata][registry-bundle-image].
```

Comment:
WDYT about a more direct approach, such as:

```suggestion
# Building OLM Bundles with SDK

The `operator-sdk build` commands have the purpose to work with Operator [bundles][registry-bundle], namely on-disk bundle directories and creating bundle [images/metadata][registry-bundle-image]
```

Reply:
The `build` subcommand is unrelated to operator bundles.